### ISSUE TITLE
Tried this out, couldn't get it working

### ISSUE THREAD

**Issue by dustinleblanc** [renoki-co/charts#13]
Hey there! Thanks for the work you've put in on this. I tried to get the demo app rolling on my local minikube cluster and ran into this error:

```
helm upgrade laravel-app \
    --install \
    --version=0.11.0 \
    renoki-co/laravel
Release "laravel-app" does not exist. Installing it now.
Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: [ValidationError(Ingress.spec.rules[0].http.paths[0].backend): unknown field "serviceName" in io.k8s.api.networking.v1.IngressBackend, ValidationError(Ingress.spec.rules[0].http.paths[0].backend): unknown field "servicePort" in io.k8s.api.networking.v1.IngressBackend, ValidationError(Ingress.spec.rules[0].http.paths[0]): missing required field "pathType" in io.k8s.api.networking.v1.HTTPIngressPath]
```

I am admittedly not awesome at Kube / Helm so it may be I am just doing something wrong. Here is my config as output by minikube:

```
😄  minikube v1.25.2 on Ubuntu 20.04 (amd64)
✨  Automatically selected the docker driver
👍  Starting control plane node minikube in cluster minikube
🚜  Pulling base image ...
🔥  Creating docker container (CPUs=2, Memory=2200MB) ...
🐳  Preparing Kubernetes v1.23.3 on Docker 20.10.12 ...
    ▪ kubelet.housekeeping-interval=5m
    ▪ Generating certificates and keys ...
    ▪ Booting up control plane ...
    ▪ Configuring RBAC rules ...
🔎  Verifying Kubernetes components...
    ▪ Using image gcr.io/k8s-minikube/storage-provisioner:v5
🌟  Enabled addons: storage-provisioner, default-storageclass
🏄  Done! kubectl is now configured to use "minikube" cluster and "default" namespace by default
```

Thanks!

**Comment by rennokki** [comment]
It seems it's an ingress misconfiguration somewhere. 🤔 I have to check this out.

**Comment by KonradMil** [comment]
I'm an absolute noob in Kube/Helm but I found probable solution on stackoverflow (https://stackoverflow.com/questions/64125048/get-error-unknown-field-servicename-in-io-k8s-api-networking-v1-ingressbacken)

TL;DR syntax has changed

rules:
    {{- range .Values.ingress.hosts }}
    - host: {{ .host | quote }}
      http:
        paths:
          {{- range .paths }}
          - path: {{ . }}
            pathType: Prefix
            backend:
              service:
                name: {{ $fullName }}
                port:
                  number: {{ $svcPort }}
          {{- end }}
    {{- end }}
  {{- end }}

**Comment by rennokki** [comment]
[Roll of v1.22 deprecated a few beta APIs](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.22.md#removal-of-several-beta-kubernetes-apis), this is why it doesnt't work.

I deployed a new version for the chart to fix it. Minimum version of Kubernetes is v1.22
